test/unit/wizards/reportcontrol.test.ts

```diff
@@ -687,4 +688,54 @@ describe('Wizards for SCL ReportControl element', () => {
       await expect(element.wizardUI.dialog).dom.to.equalSnapshot();
     }).timeout(5000);
   });
+
+  describe('define copy to other IED selector', () => {
+    let iedsPicker: FinderList;
+
+    beforeEach(async () => {
+      const sourceReportControl = doc.querySelector(
+        'IED[name="IED2"] ReportControl[name="ReportCb"]'
+      )!;
+      const wizard = reportControlCopyToIedSelector(sourceReportControl);
+      element.workflow.push(() => wizard);
+      await element.requestUpdate();
+
+      iedsPicker = <FinderList>(
+        element.wizardUI.dialog?.querySelector('finder-list')
+      );
+
+      primaryAction = <HTMLElement>(
+        element.wizardUI.dialog?.querySelector(
+          'mwc-button[slot="primaryAction"]'
+        )
+      );
+    });
+
+    it('allows to copy to multiple IED at once', () =>
+      expect(iedsPicker.multi).to.be.true);
+
+    describe('with missing sink IED', () => {
+      beforeEach(async () => {
+        iedsPicker.paths = [['IED: IED20']];
+        primaryAction.click();
+
+        await element.requestUpdate();
+      });
+
+      it('does not copy the control block ', () =>
```

Comment:
```suggestion
      it('does not copy the control block', () =>
```

Linting is my greatest skill :stuck_out_tongue_winking_eye:

Reply:
:)